dev/packages/alpha/cisco/0.0.3/dataset/asa/agent/stream/stream.yml.hbs

```diff
@@ -0,0 +1,8 @@
+paths:
+{{#each paths as |path i|}}
+  - {{path}}
+{{/each}}
+exclude_files: [".gz$"]
+tags: {{tags}}
```

Comment:
Are tags specifically used by the Cisco module? If they are generic tags if wonder if we can remove them totally for consistency with other integrations. WDYT @ruflin ?

Reply:
We will hide it in the UI and leave it as advanced options.

Reply:
@mtojek I would for now leave it up to each dataset to decide if it wants to have it exposed. I'm not sure if all inputs support it.